tests/testthat/test-zzz-e2e.R

```diff
@@ -16,10 +16,7 @@ bg$start()
 on.exit(bg$stop())
 
 r <- bg$request("POST",
-                "/repository/fetch",
-                body = jsonlite::toJSON(list(url = scalar(upstream_git))),
-                encode = "raw",
-                httr::content_type("application/json"))
+                sprintf("/repository/fetch?url=%s", upstream_git))
```

Comment:
```suggestion
                "/repository/fetch",
                query = list(url=upstream_git))
```

I believe this should work, and ensures that upstream_git is percent encoded correctly (if necessary).